### ISSUE TITLE
As a user I want to only visualize a subset of the aggregated data

### ISSUE THREAD

**Issue by ptitzler** [ibm-watson-data-lab/pixiedust#439]
Some aggregated result sets contain many rows/data points and can't be handled properly/efficiently. For example, a bar chart visualization for 200+ data points is hard to consume, even with "canvas stretch" enabled. To make it easier to analyze larger data sets I would therefore like to have the option to only visualize the topX/bottomX aggregated values.

**Comment by benhuds** [comment]
@ptitzler Can you give an example of what you envision for this?  If a dataset is larger than x then there should be a toggle on the right hand side of the display panel (like below `Cluster by`) which says `visualize top x/bottom x`?

**Comment by ptitzler** [comment]
Yes!

**Comment by vabarbosa** [comment]
@ptitzler is this something you still think should be included or do the current Filter options suffice?

**Comment by ptitzler** [comment]
The filter doesn't operate over aggregated values, right?

**Comment by vabarbosa** [comment]
you are correct. the filter is on the complete dataframe not the aggregated values.
ah ok, i know what you mean.

**Comment by ptitzler** [comment]
I think it's still valid request then, although if nobody else is asking for it at a lower priority (:
